dvc/config.py

```diff
@@ -253,19 +256,51 @@ def _load(self):
 
         self._config = None
 
+    def _load_config(self, path):
+        config = configobj.ConfigObj(path)
+        config = self._lower(config)
+        self._resolve_paths(config, path)
+        return config
+
+    @staticmethod
+    def _resolve_path(path, config_file):
+        assert os.path.isabs(config_file)
+        if os.path.isabs(path):
```

Comment:
`path.join` ignores the first path if the second one is absolute

```python
os.path.join('../dir', '/tmp/dir') # /tmp/dir
os.path.join('/hello/world', '/tmp/dir') # /tmp/dir
os.path.join('../dir', 'tmp/dir') # ../dir/tmp/dir
```

Although, I like the explicitness. :+1:

Reply:
Great catch! I'll fix it in a sec.